pkg/kubectl/cmd/apply_view_last_applied.go

```diff
@@ -140,29 +139,19 @@ func (o *ViewLastAppliedOptions) RunApplyViewLastApplied() error {
 				return err
 			}
 			fmt.Fprint(o.Out, string(jsonBuffer.Bytes()))
-		case "yaml":
+		case "yaml", "":
```

Comment:
Why we have `""`?

Reply:
Just in case the value will be an empty string.

Reply:
And this is modified based on the old method `ValidateOutputArgs`, where `""` is counted as well.

Reply:
Looks like we default to yaml on empty string, mind doing a check before the switch (maybe in the Complete method) that defaults the value of `o.OutputFormat` to `yaml` if empty?